recipes/boiledwater.html

```diff
@@ -37,5 +37,8 @@ <h3>Directions</h3>
             </ol>
         </div>
     </div>
+    <a href="index.html">
```

Comment:
Use <a href="../index.html"> to navigate to the root directory. 
Please don't move index.html into recipes folder and change the project structure.